.NET/Microsoft.Recognizers.Text.DateTime/Parsers/FullDateTimeParser.cs

```diff
@@ -489,15 +488,19 @@ public List<DateTimeParseResult> FilterResults(string query, List<DateTimeParseR
             return candidateResults;
         }
 
-        internal static void ResolveFeb29th(Dictionary<string, object> resolutionDic, string keyName, Dictionary<string, string> resolutionStr)
+        internal static void UpdateTimex(Dictionary<string, object> resolutionDic, string futureKey, string pastKey, string originTimex)
         {
-            if (!resolutionDic.ContainsKey(keyName) || !resolutionStr.ContainsKey(DateTimeResolutionKey.Timex))
+            string[] timexs = originTimex.Split('|');

```

Comment:
timexes :)
And separators must always be declared somewhere, and not used hardcoded in the code.